BlueBlazor/Components/MonacoEditor/MonacoEditor.razor.js

```diff
@@ -0,0 +1,61 @@
+﻿const MonacoProm = new Promise(async (resolve) => {
+    if (window.monaco) {
+        resolve(window.monaco)
+        return
+    }
+    if (!window.require) {
+        await new Promise((resolve) => {
+            const script = document.createElement("script")
+            script.src = "/_content/BlueBlazor/monaco-editor/min/vs/loader.js"
```

Comment:
On apps that are hosted under a sub route, this could be a problem. To make the `src` relative could avoid this.
```suggestion
            script.src = "./_content/BlueBlazor/monaco-editor/min/vs/loader.js"
```